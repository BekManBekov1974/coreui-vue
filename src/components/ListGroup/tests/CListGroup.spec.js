import { mount } from '@vue/test-utils'
import Component from '../CListGroup'

const ComponentName = 'CListGroup'
const defaultWrapper = mount(Component)
const customWrapper = mount(Component, {
  propsData: {
    flush: true
  },
  slots: {
    default: 'CListGroup content'
  }
})

describe(ComponentName, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(defaultWrapper.element).toMatchSnapshot()
  })
  it('renders correctly', () => {
    expect(customWrapper.element).toMatchSnapshot()
  })
})
