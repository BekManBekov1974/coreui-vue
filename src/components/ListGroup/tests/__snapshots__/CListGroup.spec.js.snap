// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CListGroup renders correctly 1`] = `
<div
  class="c-list-group"
/>
`;

exports[`CListGroup renders correctly 2`] = `
<div
  class="c-list-group c-list-group-flush"
>
  <template>
    CListGroup content
  </template>
</div>
`;
